docs/sources/why-use-continuous-profiling/_index.md

```diff
@@ -0,0 +1,88 @@
+
+Traditionally, profiling has been used more as an ad-hoc debugging tool. While used in many languages, particularly in Go and Java many are used to running a benchmark tool locally and getting a pprof file in go or maybe ssh'ing into a misbehaving prod instance and pulling a flamegraph from a JFR file in Java. This is great for debugging but not so great for production.
+
+Continuous profiling is a much more modern approach which is safer and more scalable for production environments. It makes use of low overhead sampling to collect profiles from production systems and stores them in a database for later analysis. This allows you to get a much more holistic view of your application and how it behaves in production.
+
+## The Core Benefits of Continuous Profiling
+
+![Diagram showing 3 benefits of continuous profiling](#)
+
+Why prioritize continuous profiling? Here are the key reasons:
+1. **In-Depth Code Insights:** It provides granular, line-level insights into how application code utilizes resources, offering the most detailed view of application performance.
+2. **Complements Other Observability Tools:** Continuous profiling fills critical gaps left by metrics, logs, and tracing, creating a more comprehensive observability strategy.
+3. **Proactive Performance Optimization:** Regular profiling enables teams to proactively identify and resolve performance bottlenecks, leading to more efficient and reliable applications.
+
+## Business Impact of Continuous Profiling
+
+![Infographic illustrating key business benefits](#)
+
+Adopting continuous profiling with tools like Pyroscope can lead to significant business advantages:
+1. **Reduced Operational Costs:** Optimization of resource usage can significantly cut down cloud and infrastructure expenses
+2. **Latency reduction:** Identifying and addressing performance bottlenecks leads to faster and more efficient applications
+3. **Enhanced Incident Management:** Faster problem identification and resolution, reducing Mean Time to Resolution (MTTR) and improving end-user experience
+
+### How Pyroscope helps cut costs
+Pyroscope's low-overhead profiling enables precise optimization of resource usage, directly impacting various cost centers in technology infrastructure. By providing in-depth insights into application performance, Pyroscope allows teams to identify and eliminate inefficiencies, leading to significant savings in areas like observability, incident management, messaging/queuing, deployment tools, and infrastructure.
+
+### How Pyroscope helps reduce latency
+Pyroscope plays a pivotal role in reducing application latency by identifying performance bottlenecks at the code level. This granular insight allows for targeted optimization, leading to faster application response times, improved user experience, and consequently, better business outcomes like increased customer satisfaction and revenue.
+
+### How Pyroscope helps with incident management
```

Comment:
graphic tbd... I usually like to show an example of when an incident is shown in metrics (i.e. cpu spike) or logs (i.e. OOM errors due to memory leak) 

and then I show how profiling tells you **why** there was a spike in cpu or **where** the memory leak is